docs/team/yuxianglim.md

```diff
@@ -0,0 +1,101 @@
+
+## Content
+1. [Overview](#overview)
+2. [Summary of Contributions](#summary-of-contributions)
+3. [Contributions to the User Guide](#contributions-to-the-user-guide)
+4. [Contributions to the Developer Guide](#contributions-to-the-developer-guide)
+
+
+## Overview
+Diet Manager is an personalized chat-bot application used for managing an individual's diet.
+
+Diet Manager is capable of tracking daily food intake and providing recommendations based on a user's specified
+health data and weight goal.
+
+## Summary of Contributions
+
+### Code Contributed
+**__Diet Manager__**:<br>
+Link to code on tP Code Dashboard 
+[**__here__**](https://nus-cs2113-ay1920s2.github.io/tp-dashboard/#breakdown=true&search=lamyuewei&sort=groupTitle&sortWithin=title&since=2020-03-01&timeframe=commit&mergegroup=false&groupSelect=groupByRepos).
+
+### Enhancements implemented
+1. Created the CheckWeightProgress class which allows users to check their past weight records from the beginning till present.
+    * The enhancement to this function is the ability to show the difference in weight of the user from the initial record to present and output message from the message bank in relative to the progress.
+    
+    * There might be further enhancement such as showing the % gain or loss in weight of the user.
+    
+    * Below is the original table which contains the sample input for user
+    
+    No. | Functions: | Descriptions: | Example: 
+    ---| --- | --- | ---
+    1  | `set-profile NAME AGE GENDER HEIGHT WEIGHT WEIGHTGOAL  ` | set user's profile data |  (eg. set-profile John 20 male 180 80 75 )
+    2  | `profile` | View user profile details | (eg. profile)
+    3  | `record-meal DATE TIME_PERIOD /FOOD_NAME -- CALORIE` | Record meal info | (eg. record-meal Monday morning /prata -- 10.0 /milk -- 6.00 )
+    4  | `check-meal DATE TIME_PERIOD` |Check meals eaten | (eg. check-meal Monday morning )
+    5  | `calculate DATE` | Calculate Calorie intake for the day | (eg.calculate Monday)
+    6  | `calculate DATE1->DATE2` | Calculate Calorie intake from DATE1 to DATE2 | (eg. calculate Monday->Wednesday )
+    7  | `list-food` | List all foods recorded in the database  | (eg. list-food )
+    8  | `addf FOOD_NAME --CALORIES` | Add a new food into database | (eg. addf beef noodles -- 7.0 )
+    9  | `delf FOOD_NAME` | Delete a food from the database | (eg. delf beef noodles )
+    10 | `new-recipe MAXIMUM_FOOD_TYPES ACTIVITY_LEVEL` | Randomly recommend recipe from database | (eg. new-recipe 2 low )
+    11 | `show-recipe` | Show the recipe recommended for user | (eg. show-recipe ) 
+    12 | `set-weight ` | Update user's weight changes | (eg. set-weight 70 )
+    13 | `check-weight-progress NAME` | Check user's weight progress | (eg. check-weight-progress John )
+    14 | `delete-weight` | Delete user's weight from progress | (eg. delete-weight 2 ) 
+    15 | `help` | Show the help function table |  (eg. help )
+    16 | `exit` | Exit application | (eg. exit )
```

Comment:
Perhaps this portion can be written in the contribution to documentation instead